src/validations/validate-parameters.ts

```diff
@@ -0,0 +1,83 @@
+import { Toolkit } from 'actions-toolkit';
+
+export interface ValidationSchema {
```

Comment:
**suggestion (if-minor):** maybe I would move this interface to a separate file to keep the files simple and reduce the logic we have, because if we have it in this current way we are going to end up having the logic about how to validate and the logic about how to define a schema in the same file, which I am not sure is the best. Anyway, this is a minor change we can always do later.

Reply:
Extracted to another file

Reply:
Nice thanks!